spec/comm.mediawiki

```diff
@@ -189,13 +192,19 @@ of their penalization.
 {|
 ! field   !! type   !! description
 |-
-| matches || &#91;object&#93; || list of [[orders.mediawiki/#Match_negotiation|Match objects]]
+| activematches || <nowiki>[object]</nowiki> || list of active [[orders.mediawiki/#Match_negotiation|Match objects]]
 |-
-| order statuses || <nowiki>[object]</nowiki> || list of <code>Order Status Object</code>
+| activeorderstatuses || <nowiki>[object]</nowiki> || list of active <code>Order Status Object</code>
 |-
 | score || int || The user's score. The math behind this is currently being adjusted
 |-
-| suspended || bool || true if suspended. will be deprecated in a future version
+| tier || int || Bond tier. One or more indicates the account can trade. Not set for legacy accounts.
+|-
+| activeBonds || <nowiki>[object]</nowiki> || list of active <code>Bond Object</code>
+|-
+| legacyFeePaid || bool || Whether the legacy fee was paid. This currently has the internal value of one bond tier. Not set for legacy servers.
+|-
+| suspended || bool || DEPRECATED. For legacy servers true if suspended.
```

Comment:
Let's also add that this also implies `tier<1`, and it just means that the user cannot trade without posting new bond.

Reply:
If the server can handle bonds, will this be set? They can't post bonds on the legacy server.

Reply:
Updated the wording.

Reply:
> If the server can handle bonds, will this be set? They can't post bonds on the legacy server.

Yes, for backward compatibility with legacy clients.

However, it's looking more and more like the release with bonds will fully kill the legacy fee, and this backward compatibility is just a transition.